wemake_python_styleguide/visitors/tokenize/comments.py

```diff
@@ -311,3 +312,24 @@ def _post_visit(self) -> None:
             self.add_violation(
                 OveruseOfNoqaCommentViolation(text=str(self._noqa_count)),
             )
+
+
+@final
+class CommentInFormattedStringVisitor(BaseTokenVisitor):  # pragma: >=3.12 cover
+    """Checks comment in formatted strings."""
+
+    _comment_in_fstring: ClassVar[re.Pattern[str]] = re.compile(
+        r'.*f[\"\'].*\{[^}]*#[^}]*$', re.MULTILINE
+    )
```

Comment:
Please, make this regex multiline and add comments. Right now it is hard to read. See `MultilineFormattedStringViolation` for the example.